docs/appendices/environment_vars.rst

```diff
@@ -143,6 +146,8 @@ There are several environment variables that affect the way Toil runs.
 |                                  | jobs.                                              |
 |                                  | There is no default value for this variable.       |
 +----------------------------------+----------------------------------------------------+
+| TOIL_SLURM_TIME                  | Slurm job time limit, in [DD-]HH:MM:SS format.     |
```

Comment:
An example might be useful (literally adding `e.g. TOIL_SLURM_TIME="04:00:00" for 4 hours`, assuming that this produces a 4 hour time limit), or a link to SLURM docs if this is a direct alias for a slurm arg.